angularSTEP/src/app/converter.ts

```diff
@@ -19,6 +19,7 @@ export class Converter {
         wishlist: user.wishlist,
         shoppingList: user.shoppingList,
         following: user.following,
+        numRecipes: user.numRecipes,
```

Comment:
I'm curious what purpose this serves, since it seems like `recipes.length` would effectively get the same result in all cases(?) without the possibility of the two getting out of sync

Reply:
Yes, at first we had wanted to be able to sort users by the number of recipes they had. Currently this is not a priority so we actually will not need this.